.gitignore

```diff
@@ -21,3 +21,13 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# IDE Files/Directories
+.idea/
```

Comment:
might be worth including VSCode's too

Reply:
What files/directories should i put for VSCode? I was going to add it, but I couldn't remember which ones there were since I haven't used it in so long.